.cookiecutter.json

```diff
@@ -11,5 +11,5 @@
             "src/h_pyramid_sentry/__init__.py"
         ]
     },
-    "_template": "git@github.com:hypothesis/h-cookiecutter-pypackage.git"
+    "_template": "gh:hypothesis/h-cookiecutter-pypackage"
```

Comment:
Looks like this refers to https://github.com/hypothesis/hdev/blob/9932d1e08647505302a8de94539ed3450fd9bb1e/src/hdev/model/project.py#L103?